src/query/ast/src/ast/format/syntax/dml.rs

```diff
@@ -0,0 +1,202 @@
+            RcDoc::text("INTO")
+        })
+        .append(
+            RcDoc::line()
+                .nest(NEST_FACTOR)
+                .append(if let Some(catalog) = insert_stmt.catalog {
+                    RcDoc::text(catalog.to_string()).append(RcDoc::text("."))
+                } else {
+                    RcDoc::nil()
+                })
+                .append(if let Some(database) = insert_stmt.database {
+                    RcDoc::text(database.to_string()).append(RcDoc::text("."))
+                } else {
+                    RcDoc::nil()
+                })
+                .append(RcDoc::text(insert_stmt.table.to_string()))
+                .append(if !insert_stmt.columns.is_empty() {
+                    RcDoc::space()
+                        .append(RcDoc::text("("))
+                        .append(inline_comma(
+                            insert_stmt
+                                .columns
+                                .into_iter()
+                                .map(|ident| RcDoc::text(ident.to_string())),
+                        ))
+                        .append(RcDoc::text(")"))
+                } else {
+                    RcDoc::nil()
+                }),
+        )
+        .append(pretty_source(insert_stmt.source))
+}
+
+fn pretty_source(source: InsertSource) -> RcDoc {
+    RcDoc::line().append(match source {
+        InsertSource::Streaming {
+            format,
+            rest_tokens,
+        } => RcDoc::text("FORMAT")
+            .append(RcDoc::space())
+            .append(RcDoc::text(format))
+            .append(
+                RcDoc::line().nest(NEST_FACTOR).append(RcDoc::text(
+                    (&rest_tokens[0].source[rest_tokens.first().unwrap().span.start
+                        ..rest_tokens.last().unwrap().span.end])
+                        .to_string(),
+                )),
+            ),
+        InsertSource::Values { rest_tokens } => RcDoc::text("VALUES").append(
+            RcDoc::line().nest(NEST_FACTOR).append(RcDoc::text(
+                (&rest_tokens[0].source[rest_tokens.first().unwrap().span.start
+                    ..rest_tokens.last().unwrap().span.end])
+                    .to_string(),
+            )),
+        ),
+        InsertSource::Select { query } => pretty_query(*query),
+    })
+}
+
+pub(crate) fn pretty_delete<'a>(
+    table: TableReference<'a>,
+    selection: Option<Expr<'a>>,
+) -> RcDoc<'a> {
+    RcDoc::text("DELETE FROM")
+        .append(RcDoc::line().nest(NEST_FACTOR).append(pretty_table(table)))
+        .append(if let Some(selection) = selection {
+            RcDoc::line().append(RcDoc::text("WHERE")).append(
+                RcDoc::line()
+                    .nest(NEST_FACTOR)
+                    .append(pretty_expr(selection).nest(NEST_FACTOR).group()),
+            )
+        } else {
+            RcDoc::nil()
+        })
+}
+
+pub(crate) fn pretty_copy(copy_stmt: CopyStmt) -> RcDoc {
+    RcDoc::text("COPY INTO")
```

Comment:
Please place the `INFO` in the following line to align with `FROM`.

```sql
COPY
    FROM 
        s3://xxxx
    INTO
        mytable
```

Reply:
`COPY` is aligned with `FROM`, `INTO` is only one word, it is no need to put it on a separate line.

for example:

```sql
COPY INTO
    t1
FROM
    's3://mybucket/data.csv'
FILE_FORMAT = (
    field_delimiter = ",",
    record_delimiter = "\n",
    skip_header = "1",
    type = "CSV"
)
SIZE_LIMIT = 10
```

Reply:
We supports into stages. So into could be s3://abc/def too.

Reply:
ok

Reply:
done.